app/models/host/managed.rb

```diff
@@ -271,8 +271,12 @@ def built(installed = true)
     self.installed_at = Time.now.utc if installed
 
     if self.save
-      recipients = owner ? owner.recipients_for(:host_built) : []
-      MailNotification[:host_built].deliver(self, :users => recipients) if recipients.present?
+
+      if installed
```

Comment:
I guess, I don't think it buys you much.  I moved it but I did not test it in isolation, it belongs as a private method IMHO.